tests/yam/agama/boot_agama.pm

```diff
@@ -52,12 +52,19 @@ sub run {
     my $grub_menu = $testapi::distri->get_grub_menu_agama();
     my $grub_entry_edition = $testapi::distri->get_grub_entry_edition();
     my $agama_up_an_running = $testapi::distri->get_agama_up_an_running();
+    my $profile_url;
 
     # prepare kernel parameters
     if (my $agama_auto = get_var('AGAMA_AUTO')) {
-        my $path = expand_agama_profile($agama_auto);
-        set_var('AGAMA_AUTO', $path);
-        set_var('EXTRABOOTPARAMS', get_var('EXTRABOOTPARAMS', '') . " agama.auto=\"$path\"");
+        # Currently jsonnet is only available to install on x86_64, remove this after include in the rest of architectures.
+        if ($agama_auto eq "1" && (is_x86_64() || is_aarch64())) {
```

Comment:
you don't need to do this explicitly in Perl, just use the variable without `eq "1"`.

Reply:
agreed to do a regex instead of checking arch.